react/features/chat/components/web/ChatMessage.js

```diff
@@ -79,13 +79,14 @@ class ChatMessage extends AbstractChatMessage<Props> {
     _renderDisplayName() {
         const senderHasSuperHeroAddress = Boolean(this.props.message.akAddress)
            || isAccountOrChainName(this.props.message.displayName);
+        const localParticipantHasSuperHeroWallet = Boolean(this.props.hasWallet);
```

Comment:
you don't add it to the redux mapStateToProps. And I already done that at other branch.

Reply:
localParticipantHasSuperHeroWallet is a better name